src/Aquifer.API/Endpoints/Resources/Content/NeedsTranslation/List/Endpoint.cs

```diff
@@ -0,0 +1,139 @@
+                    INNER JOIN Resources R ON R.Id = RC.ResourceId
+                    INNER JOIN ParentResources PR ON PR.id = R.ParentResourceId
+                    INNER JOIN Languages L ON L.Id = RC.LanguageId
+                    INNER JOIN ResourceContentVersions RCV ON RCV.ResourceContentId = RC.Id
+                WHERE RC.MediaType != {(int)ResourceContentMediaType.Audio}
+                {ApplyLanguageIdFilter(user.LanguageId ?? 1)}
+                {ApplyParentResourceIdFilter(req.ParentResourceId)}
+                {ApplySearchQueryFilter(hasSearchQuery, isExactSearch)}
+                {ApplyBookAndChapterFilter(req.BookCode, req.StartChapter, req.EndChapter)}
+                {(getTotalCount ? "" : groupingWithOffset)}
+                """;
+    }
+
+    private static string ApplyLanguageIdFilter(int languageId)
+    {
+        return $"""
+                AND RC.LanguageId = 1 AND RCV.IsPublished = 1 AND NOT EXISTS (
+                    SELECT 1 FROM ResourceContents RCL
+                    WHERE RCL.ResourceId = R.Id AND RCL.LanguageId = {languageId}
+                )
+                """;
+    }
+
+    private static string ApplyParentResourceIdFilter(int? parentResourceId)
+    {
+        return parentResourceId.HasValue ? $"AND R.ParentResourceId = {parentResourceId.Value}" : "";
+    }
+
+    private static string ApplyBookAndChapterFilter(string? bookCode, int? startChapter, int? endChapter)
+    {
+        var verseRange = BibleUtilities.VerseRangeForBookAndChapters(bookCode, startChapter, endChapter);
+        if (verseRange is null)
+        {
+            return "";
+        }
+
+        var startVerseId = verseRange.Value.Item1;
+        var endVerseId = verseRange.Value.Item2;
+
+        return $"""
+                AND (
+                EXISTS (SELECT 1
+                    FROM [VerseResources] AS [v]
+                    WHERE [R].[Id] = [v].[ResourceId] AND [v].[VerseId] >= {startVerseId} AND [v].[VerseId] <= {endVerseId})
+                OR
+                EXISTS (SELECT 1
+                    FROM [PassageResources] AS [p]
+                    INNER JOIN [Passages] AS [p0] ON [p].[PassageId] = [p0].[Id]
+                    WHERE [R].[Id] = [p].[ResourceId] AND [p0].[StartVerseId] >= {startVerseId} AND [p0].[StartVerseId] <= {endVerseId})
+                OR
+                EXISTS (SELECT 1
+                    FROM [PassageResources] AS [p1]
+                    INNER JOIN [Passages] AS [p2] ON [p1].[PassageId] = [p2].[Id]
+                    WHERE [R].[Id] = [p1].[ResourceId] AND [p2].[EndVerseId] >= {startVerseId} AND [p2].[EndVerseId] <= {endVerseId})
+                OR
+                EXISTS (SELECT 1
+                    FROM [PassageResources] AS [p3]
+                    INNER JOIN [Passages] AS [p4] ON [p3].[PassageId] = [p4].[Id]
+                    WHERE [R].[Id] = [p3].[ResourceId] AND [p4].[StartVerseId] <= {startVerseId} AND [p4].[EndVerseId] >= {endVerseId})
+                )
```

Comment:
I'm guessing that you'll see similar or possibly better performance if you combine these three SELECTS into one SELECT with OR clauses in the WHERE condition.  I also think it's easier to understand.
```suggestion
                EXISTS
                (
                    SELECT 1
                    FROM [PassageResources] AS [p]
                    INNER JOIN [Passages] AS [p0] ON [p].[PassageId] = [p0].[Id]
                    WHERE [R].[Id] = [p].[ResourceId] AND
                    (
                        [p0].[StartVerseId] BETWEEN {startVerseId} AND {endVerseId} OR
                        [p0].[EndVerseId] BETWEEN {startVerseId} AND {endVerseId} OR
                        ([p0].[StartVerseId] <= {startVerseId} AND [p0].[EndVerseId] >= {endVerseId})
                    )
                )
```